src/Emoji/README.md

```diff
@@ -0,0 +1,23 @@
+# < Emoji />
+
+## Props:
+
+```ts
+symbol:   string;
```

Comment:
Should we clarify that you have to add (or copy and paste) the emoji in directly as a string rather than it's text description?

Reply:
Good point